### PR TITLE
fix(theme): fix header color on Customise Drawer page

### LINKED ISSUE #189: Font color problem with Customise Drawer page
- Set theme to Vlad
- Go to Customise Drawer options page
- See that the `Customise Drawer` heading does not have the heading font color, but the `Visible Apps` button does have the color

### PR DESCRIPTION
**Pull request by hayribakici**
This PR fixes #189

### CODE REVIEW

File: app/src/main/res/layout/customize_app_drawer_fragment.xml [hunk 1]
```diff
@@ -14,10 +14,9 @@
         android:id="@+id/customize_app_drawer_fragment_back"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="32dp"
-        android:text="@string/customize_app_drawer_fragment_visible_apps"
-        android:textAppearance="@style/TextAppearance.AppCompat"
-        android:textSize="@dimen/font_size_customize_options"
+        android:padding="@dimen/_8ssp"
+        android:paddingStart="0dp"
+        app:layout_constraintTop_toTopOf="parent"
```
Comment (TobiTenno) [inline app/src/main/res/layout/customize_app_drawer_fragment.xml:19]: these are in-line w/ da163e7d31510131348ef5b7bc7100c579a00060

Comment (TobiTenno) [reply app/src/main/res/layout/customize_app_drawer_fragment.xml:19]: @hayribakici after the merge/update, i think your change was in a different place, possibly, in the same file

**Review by jkuester — APPROVED** [review]
LGTM!

**Comment by jkuester** [comment]
I love how all three of us are getting credited for this 2 line change... :laughing: :facepalm:

**Pull request merged**